### ISSUE TITLE
How to crop/cut?

### ISSUE THREAD

**Issue by vans163** [cocoa-xu/evision#53]
In python its
```
import cv2
img = cv2.imread("lenna.png")
crop_img = img[y:y+h, x:x+w]
cv2.imshow("cropped", crop_img)
cv2.waitKey(0)
```

**Comment by cocoa-xu** [comment]
No direct way to do this at the moment. To crop/cut an image you need to use one or multiple functions from Nx (which also means the image will need to be converted to an Nx tensor first).

**Comment by cocoa-xu** [comment]
A similar function (and other image manipulation operations) will be considered and implemented in #48.

**Comment by cocoa-xu** [comment]
This issue will be tracked in #48.
